Backend/src/main/java/com/valantic/myplaylist/service/AuthenticationService.java

```diff
@@ -0,0 +1,54 @@
+package com.valantic.myplaylist.service;
+
+
+import com.valantic.myplaylist.model.AuthenticationDTO;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.http.converter.HttpMessageNotReadableException;
+import org.springframework.stereotype.Service;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.client.RestTemplate;
+
+@Service
+public class AuthenticationService {
+
+    @Value("${com.valantic.playlist.client-id}")
+    private String username;
+    @Value("${com.valantic.playlist.client-secret}")
+    private String password;
+
+    private final RestTemplate restTemplate;
+    private final String authenticationAPI = "https://accounts.spotify.com/api/token";
```

Comment:
Kannst du als Konstante definieren.
(Beachte dabei auch die Namenskonvention für Konstanten in Java)

Reply:
das war ein Vorschlage von intelliJ
habe nun wieder als Variabel definiert